README.md

```diff
@@ -1 +1 @@
-# Code-review-template
```

Comment:
Please consider using the Microverse READEME.md template found [here](https://github.com/microverseinc/readme-template#about-projectl) in order to have more professional documentation 📝

Reply:
Great job adding an inline comment and including the resource where the template can be found, please I'd suggest you give an elaborate reason why the student needs to document his project 😄